### PR TITLE
Mobile menu js query

### PR DESCRIPTION
**Pull request by juanmanuelbyc**
Create a dropdown mobile menu and apply some js query to make it responsive to user interaction

The most important project requirements are:

When the user clicks (or taps) the hamburger button, the mobile menu appears.
When the user clicks (or taps) the close (X) button, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const closeBtn = document.querySelector('.close-btn');
+const mobilenavlinks = document.querySelectorAll('.m-navlink');
+
+btn.addEventListener('click', () => {
+  document.getElementById('mobile-menu').style.visibility = 'visible';
+});
+
+closeBtn.addEventListener('click', () => {
+  document.getElementById('mobile-menu').style.visibility = 'hidden';
+});
```
Comment (omarsalem7) [inline js/index.js]: - [ ] Kindly make your background match the one in the Figma by making the background blur when clicking on the mobile menu.

| Your design  | Figma design|
| ------------- | ------------- |
| image | image|

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @juanmanuelbyc, @Bateyjosue 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️
- Open/close menu work properly ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  sections.forEach((sect) => {
+    sect.style.filter = 'none';
+  });
+});
+
+mobilenavlinks.forEach((el) => {
+  el.addEventListener('click', () => {
+    document.getElementById('mobile-menu').style.visibility = 'hidden';
+    sections.forEach((sect) => {
+      sect.style.filter = 'none';
```
Comment (juanmanuelbyc) [inline js/index.js:24]: Added a blur filter to my sections when the mobile menu is displayed and turned it off when the menu disappears

File: js/index.js (hunk 1 above)
Comment (juanmanuelbyc) [reply js/index.js]: Thanks for the feedback! Please check my comment below. Regards.

File: index.html [hunk 3]
```diff
@@ -9,17 +9,24 @@
...
         <a class="my-logo" href="#">My Logo</a>
         <div class="navlist">
           <a class="navlink" href="#portfolio">Portfolio</a>
           <a class="navlink" href="#about">About</a>
           <a class="navlink" href="#contactme">Contact</a>
-          <a class="menu-icon" href="#"></a>
+          <a id="hamburguer-btn" class="menu-icon" href="#"></a>
         </div>
       </nav>
     </header>
```
Comment (algerina) [inline index.html:32]: - [ ] Your hamburger menu is not visible on the mobile version, kindly try to fix it.
- [ ] The social media icons are not displaying properly as you can see:

rev1

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @juanmanuelbyc , 

You've done an amazing job so far fir 👏 💯 👍
There are some issues that you still need to work on to go to the next project but you are almost there! 

## Highlights
  -  Linters are passing :heavy_check_mark: 
  -  Your project looks good :heavy_check_mark: 
 

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.


You can also consider:
  - [x] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 3 above)
Comment (juanmanuelbyc) [reply index.html:32]: Untitled
Please make sure for your review you set the mobile device width to 375px (design width according to the Figma template) or set a specific device like samsumg s8

Comment (juanmanuelbyc) [reply index.html:32]: the icons issue must be on your side.

**Review by omarsalem7 — APPROVED** [review]
Hi @juanmanuelbyc 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**